src/bootstrap/lib.rs

```diff
@@ -498,6 +493,15 @@ impl Build {
             tool_artifacts: Default::default(),
         };
 
+        let bootstrap_out = &build.bootstrap_out;
+        if bootstrap_out.join("bootstrap").exists() {
+            let bin = bootstrap_out.join("bootstrap");
+            build.copy(&bin, &bootstrap_out.join("rustc"));
+            build.copy(&bin, &bootstrap_out.join("rustdoc"));
+            build.copy(&bin, &bootstrap_out.join("sccache-plus-cl.exe"));
```

Comment:
I'm a little surprised to see only this binary with an .exe extension here -- shouldn't they all gain it (presumably actually https://doc.rust-lang.org/nightly/std/env/consts/constant.EXE_SUFFIX.html)?

(And, do we need to check to copy bootstrap.exe and not just bootstrap)?